### ISSUE TITLE
mobileAutoEnable totally breaks Chrome's touch events

### ISSUE THREAD

**Issue by kamkha** [goldfire/howler.js#1003]
Now that Howler is running its event-based auto-enable on Chrome, all touch events are broken due to Chrome forcing document-level touch handlers to be _passive_.

![image](https://user-images.githubusercontent.com/742860/43023428-dc678fdc-8c38-11e8-9d2b-5d1efd118199.png)
(Clickable link from the image: https://www.chromestatus.com/features/5093566007214080)

To reproduce:
1. Load any page that includes howler.js 2.0.14 and initializes a Howl object in Chrome, like any of the demos on https://howlerjs.com/.
2. If on desktop, enter mobile-emulation mode: open the DevTools (<kbd>⌘ ⌥ i</kbd> on Mac) and then the device toolbar (<kbd>⌘ ⇧ m</kbd>), and ensure "Mobile" is selected as the "Device type" (so that the typical mouse events are instead emulated touch events).
3. Try to interact with the document by clicking on any element, scrolling with the cursor, etc. — nothing will happen.

Howler's document-level `touchstart` event handler calls `preventDefault` on the event, but recent versions of Chrome force `touchstart` event handlers — and other potentially scroll-blocking handlers — to be passive (see [the relevant item in the Chrome platform changelog](https://www.chromestatus.com/features/5093566007214080)). Passive events can't have their default actions prevented, so [Howler's `unlock` handler](https://github.com/goldfire/howler.js/blob/b8591f0e32228bb05bd346d15a1a302d3ebc8017/src/howler.core.js#L306) throws an error which breaks that touch event and prevents interaction with the document.

This is affecting Chrome desktop and Chrome for Android version 56 and above, including web views on Android.

A quick fix would involve checking that `unlock` handler to make sure it isn't passive before preventing its default handler. However, it's unclear to me why the default handler should be prevented at all, as that `unlock` method seems like it should _always_ be passive — perhaps that line can be removed altogether?

**Comment by benwick** [comment]
If the [line](https://github.com/goldfire/howler.js/blob/master/src/howler.core.js#L306) serves no purpose, we should remove it. Because it breaks the opening of links in chrome... #1011

**Comment by FiveHundredMiles** [comment]
I encountered this error in my app after I updated the library to v2.0.14 yesterday.
But, except for the error message, it looks like everything is still functioning on my app (the audio part).

Will anyone be kind enough to share the code on how can I implement a quick fix to get rid  of the error message? Something we can just cut and paste the code - because some of us like me don't really understand how the audio part works. :-) Thank you so much in advance.
